examples/evse-app/linux/README.md

```
@@ -33,11 +33,27 @@ details.
+
+-   Ensure your repo is up to date with submodules fetched
+
+                $ cd connectedhomeip
+                $ ./scripts/checkout_submodules.py --platform linux --recursive
+
+                # Alternatively you can check out all submodules and resync with:
+                $ git submodule sync --recursive; git submodule update --init --recursive
+
+-   Activate your shell (do this every time you open the new terminal window)
+
+                $ cd ~/connectedhomeip
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)